projects/knora-ui/src/lib/viewer/values/list-value/list-value.component.ts

```diff
@@ -6,22 +6,20 @@ import {
   UpdateListValue,
   ListNodeV2,
   KnoraApiConnection,
-  ApiResponseError, ResourcePropertyDefinition
```

Comment:
you can safely remove `NgZone` from the imports since it is unused now

Reply:
removed in a18c72d